include/api/CAnomalyJob.h

```diff
@@ -412,8 +412,8 @@ class API_EXPORT CAnomalyJob : public CDataProcessor {
                                               const std::string& partitionFieldValue,
                                               model::CResourceMonitor& resourceMonitor);
 
-    //! Prune all the models
-    void pruneAllModels();
+    //! Prune all the models that exceed \p buckets in age
```

Comment:
There should be an extra comment to say that 0 has a special value, meaning "so old they're effectively dead".

(It's worth noting there's a discontinuity here, as 1 is more aggressive than 2, 2 is more aggressive than 3, 3 is more aggressive than 4, 0 less aggressive than 1000.)